DevTools/HackStudio/Editor.h

```diff
@@ -42,6 +44,12 @@ class Editor final : public GUI::TextEditor {
     EditorWrapper& wrapper();
     const EditorWrapper& wrapper() const;
 
+    Vector<size_t> breakpoint_lines() const { return m_breakpoint_lines; }
```

Comment:
This creates a copy of the vector on every call. That's probably not what we want. Instead return a `const Vector&`
```suggestion
    const Vector<size_t>& breakpoint_lines() const { return m_breakpoint_lines; }
```